static/src/inline-svgs/icon/share-facebook.svg

```diff
@@ -0,0 +1 @@
+<svg width="32" height="32" viewBox="-2 -2 32 32"><path d="M17.9 14h-3v8h-2.9v-8h-2v-2.9h2v-2.4c0-1.9 1.1-3.7 4-3.7 1.2 0 2 .1 2 .1v3h-1.8c-1 0-1.2.5-1.2 1.3v1.8h3l-.1 2.8z"/></svg>
```

Comment:
No point leaving them in the CSS for everyone if they're only needed in IE8. Perhaps we can move them into the old-ie stylesheet?